python/lsst/ts/salobj/base_script.py

```diff
@@ -227,6 +230,12 @@ def checkpoints(self):
         """
         return self.evt_checkpoints.data
 
+    @property
+    def group_id(self):
+        """Get the group ID, or "" if not set.
```

Comment:
It seems that most properties do document the return type but this one is missing.

Reply:
I only document the return type when it is not obvious, but perhaps this one fits that category. I'll add something.